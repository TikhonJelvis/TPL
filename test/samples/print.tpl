print 10
print "foo"
print {}
print null
print true
print false
print [1,2,3]
