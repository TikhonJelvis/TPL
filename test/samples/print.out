10
foo
{...}
null
true
false
[1, 2, 3]
null
